### DISCUSSION TITLE
Adding new subscription, how to NOT get 15 past items show up?

### DISCUSSION THREAD

**Discussion by lausvi** [FreshRSS/FreshRSS#3866] [Q&A]
Hi, 

everytime I add a new subscription, I get 15 of the latest items from that subscription added to my main stream. How can I add a new subscription, and only get the latest / only the future posts on that subscription to show up? I don't want to mess up my main stream everytime I add a new subscription.

**Comment by Alkarex** [comment]
Hello,
At the moment, there is no option for that besides marking those items as read. When we get global filters, it will be possible to automatically mark whatever as read.
For now, you can select all old publication dates with a search expression such as `!pubdate:P30d` and mark them all as read.
